apps/editing-toolkit/editing-toolkit-plugin/wpcom-coming-soon/styles.scss

```diff
@@ -0,0 +1,166 @@
+@import '~@automattic/typography/styles/fonts';
+
+html {
+	/* No admin bar nor marketing bar on this page */
+	margin-top: 0 !important;
+}
+.body {
+	background: #117ac9;
+	color: #fff;
+	display: grid;
+	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu',
```

Comment:
You can also pull this from https://github.com/Automattic/wp-calypso/blob/f9632e9eeeca5427f14d497c00a46c3fad9f04d2/packages/typography/styles/variables.scss#L8-L9